Sources/StreamChat/Database/DTOs/ChannelReadDTO.swift

```diff
@@ -7,7 +7,7 @@ import Foundation
 
 @objc(ChannelReadDTO)
 class ChannelReadDTO: NSManagedObject {
-    @NSManaged var lastReadAt: Date?
```

Comment:
Are we sure about this? We have some problems with non-optional dates in our DTOs. Won't this cause any unwanted crashes?

Reply:
Yep, it's safe. I made the field non-optional in DB schema and provided the default value.